src/board/UBBoardView.cpp

```diff
@@ -1924,3 +1924,13 @@ bool UBBoardView::hasSelectedParents(QGraphicsItem * item)
         return false;
     return hasSelectedParents(item->parentItem());
 }
+
+void UBBoardView::releaseAllInputDevices()
+{
+    if (mMouseButtonIsPressed || mTabletStylusIsPressed || mPendingStylusReleaseEvent)
+    {
+        mMouseButtonIsPressed = false;
+        mTabletStylusIsPressed = false;
+        mPendingStylusReleaseEvent = false;
+    }
```

Comment:
I think the `if` is not really necessary here. The result is anyway to set all three variables to `false` and there is no performance benefit to test them first.